test cases/common/224 generator capture/cat.py

```diff
@@ -0,0 +1,6 @@
+#! /usr/bin/env python3
+
+import sys
+with open(sys.argv[1], 'r') as f:
+    for line in f:
+         print(line)
```

Comment:
__[Flake8]__ [E111] indentation is not a multiple of four ([link](https://sider.review/gh/repos/19784232/pulls/5573#issue-4288102))

<sub>posted by Sider</sub>